packages/elements-react/src/locales/de.json

```diff
@@ -260,6 +260,20 @@
   "property.phone": "Telefon",
   "property.code": "Code",
   "property.username": "Benutzername",
+  "consent.title": "",
+  "consent.subtitle": "",
+  "consent.scope.openid.title": "",
+  "consent.scope.openid.description": "",
+  "consent.scope.offline_access.title": "",
+  "consent.scope.offline_access.description": "",
+  "consent.scope.profile.title": "",
```

Comment:
Can you use gpt to translate the new text into the different languages so we have i18n for them?